go.mod

```diff
@@ -3,33 +3,68 @@ module github.com/kinvolk/terraform-provider-tinkerbell
 go 1.14
 
 require (
+	cloud.google.com/go v0.73.0 // indirect
+	code.gitea.io/sdk/gitea v0.13.2 // indirect
+	github.com/Azure/azure-storage-blob-go v0.12.0 // indirect
```

Comment:
Any idea where these indirects came from? Is it goreleaser?
If so, we should be able to avoid installing goreleaser in a way that affects this go.mod, even without a separate GH action - although, that may be an option too.

Reply:
Yeah, this is because of goreleaser mostly. I don't really like the approach with `tools.go` either. Let me create an issue for it.

Reply:
Created  #21. I could replace `tools.go` with `go run -mod=mod`, but then we use tools version of whatever someone has installed in their GOPATH.

Reply:
Pushed https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/20/commits/1d2377da4208ffc773fe6738b3ff2d08c3c9a317 which seems to work fine.